tests/Functional/Admin/SharedBlockAdminTest.php

```diff
@@ -144,20 +144,6 @@ public static function provideFormUrlsCases(): iterable
         ]];
 
         // Blocks From SonataPageBundle
-        yield 'Create Shared Block - Children Pages' => ['/admin/tests/app/sonatapageblock/shared/create', [
-            'uniqid' => 'shared_block',
-            'type' => 'sonata.page.block.children_pages',
-        ], 'btn_create_and_list', [
-            'shared_block[name]' => 'Name',
-            'shared_block[enabled]' => 1,
-            'shared_block[settings][title]' => 'Title',
-            'shared_block[settings][translation_domain]' => 'SonataPageBundle',
-            'shared_block[settings][icon]' => 'fa fa-home',
-            'shared_block[settings][current]' => 1,
-            // 'shared_block[settings][pageId]' => 1,
```

Comment:
Fixed this one, This blocks currently cannot be used as a shared block, in order to be able to select a pageId you need to be in the context, of the blockAdmin as a child of the pageAdmin